test/diameter_nasreq_SUITE.erl

```diff
@@ -57,14 +57,16 @@
 		    #{handler => 'ergw_aaa_nasreq'}},
 	  apps =>
 	      #{default =>
+		    #{'Origin-Host' => <<"dummy.host">>,
+		      procedures =>
 		    #{init => [#{service => <<"Default">>}],
 		      authenticate => [#{service => <<"NASREQ">>}],
 		      authorize => [#{service => <<"NASREQ">>}],
 		      start => [#{service => <<"NASREQ">>}],
```

Comment:
the content of `procedures` needs to be reindent.

Reply:
ok